### DISCUSSION TITLE
possible bug report: 0 or -1 for base

### DISCUSSION THREAD

**Discussion by ghost** [bulletphysics/bullet3#3778] [General]
Hi there,
might be a bug, in the guide in createMultiBody it states "0 for the base" in linkParentIndices and here the links actually seem to start with 0 instead of -1 like with the other linkIndex parameters.
I guess the numbering is shifted by 1.
Regards

**Comment by erwincoumans** [comment]
It is a bit complicated, the URDF link indices start with 0 while PyBullet native 'link index' starts at -1 for base, and 0 for links. Need to look into it later.

**Comment by ghost** [comment]
I think it's best to use loadURDF.

erwincoumans <notifications@github.com> schrieb am Do., 21. Mai 2020, 18:53:
